images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -226,15 +226,14 @@ function Get-DotnetRuntimes {
 
 function Get-DotnetFrameworkTools {
     $path = "${env:ProgramFiles(x86)}\Microsoft SDKs\Windows\*\*\NETFX*"
-    $frameworkVersions = @()
-    Get-ChildItem -Path $path -Directory | ForEach-Object {
-        $frameworkVersions += ($_.Name -Split(" "))[1]
-        $frameworkPath = $_.Fullname -Replace " \d+\.\d+(\.\d+)?", " <version>"
-    }
+    Get-ChildItem -Path $path -Directory | Group-Object {
+        $_.Fullname -Replace " \d+\.\d+(\.\d+)?", " <version>"
+    } | ForEach-Object {
         [PSCustomObject]@{
-            Versions = $frameworkVersions -Join " "
-            Path = $frameworkPath
+            Versions =  $_.Group | ForEach-Object { ((Get-Item $_).Name -Split(" "))[1] }
```

Comment:
No need to use Get-Item: `$_.Name.Split(" ")[1]`

Reply:
Thank you @al-cheb